posix-mapper/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM images.opencadc.org/library/cadc-tomcat:1
+
+COPY build/libs/posix-mapper.war /usr/share/tomcat/webapps/posix.war
```

Comment:
The resulting war name needs to also be posix-mapper.war so that people can correctly know how to use `war-rename.conf` if they decide they want to.

Reply:
Ah, I see.  I missed that.